specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3562,6 +3562,462 @@
+          }
+        },
+        "tags": [
+          "ThreatIntelligence"
+        ],
+        "description": "Collect the threat intelligence metrics.",
+        "operationId": "ThreatIntelligenceIndicator_CollectMetrics",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/ThreatIntelligenceIndicatorEntityKind"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ThreatIntelligenceMetric"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/bulkDelete": {
+      "post": {
+        "x-ms-examples": {
+          "Bulk delete the threat intelligence entities.": {
+            "$ref": "./examples/threatintelligence/BulkDeleteThreatIntelligence.json"
+          }
+        },
+        "tags": [
+          "ThreatIntelligence"
+        ],
+        "description": "Bulk delete the threat intelligence entities.",
+        "operationId": "ThreatIntelligenceIndicators_BulkDelete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/ThreatIntelligenceBulkRequestBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/bulkTag": {
```

Comment:
leverage ARM batch API. what is this API enabling?

Reply:
This api is enabling bulk update of items (Indicators in our scenario) with new Tags (field)

Reply:
Updated.

Reply:
Note for other reviewers: Their scenario requires separate RBAC to update tags. The only other alternative would be to integrate the authorization SDK and that is not an easy burden. I'm approving this.

---
In reply to: [482086628](https://github.com/Azure/azure-rest-api-specs/pull/10463#discussion_r482086628) [](ancestors = 482086628)